pubspec.yaml

```diff
@@ -27,19 +27,16 @@ dev_dependencies:
   flutter_lints: ^1.0.0
 
 flutter_native_splash:
-  background_image: lib/assets/background.png
-  image: lib/assets/logo.png
+  background_image: assets/images/logos_and_icons/background.png
+  image: assets/images/logos_and_icons/logo.png
 
 flutter:
-  uses-material-design: true
-
   # To add assets to your application, add an assets section, like this:
   assets:
-    - lib/assets/
-    - images/
-    - images/beasties/blob1.png
-    - images/beasties/
-  #   - images/a_dot_ham.jpeg
+    - assets/images/
+    - assets/images/beasties/
```

Comment:
Can we just do something like assets/images/*? Or does that not work

Reply:
That doesn't work in flutter unfortunately :(